docs/babel.rst

```diff
@@ -0,0 +1,67 @@
+Translations
+============
+One key goal of ``pyhf`` is to provide seamless translations between other statistical frameworks and ``pyhf``.
+This page details the various ways to translate from a tool you might already be using as part of an existing analysis to ``pyhf``.
+Many of these solutions involve extracting out the ``HistFactory`` workspace and then running `pyhf xml2json <cli.html#pyhf-xml2json>`_ which provides a single JSON workspace that can be loaded directly into ``pyhf``.
+
+HistFitter
+----------
+
+In order to go from `HistFitter` to `pyhf`, the first step is to extract out the `HistFactory` workspaces. Assuming you have an existing configuration file, ``config.py``, you likely run an exclusion fit like so:
```

Comment:
As the double quotes are being used above for pyhf and HistFactory they should be used throughout. Since I asked for this but didn't change it anywhere I will go ahead and make this change.

Reply:
Done.